metadata_service/__init__.py

```diff
@@ -73,7 +73,7 @@ def create_app(*, config_module_class: str) -> Flask:
     api.add_resource(TableDescriptionAPI,
                      '/table/<path:table_uri>/description')
     api.add_resource(TableTagAPI,
-                     '/table/<path:table_uri>/tag',
+                     '/table/<path:table_uri>/tag/',
```

Comment:
Would introducing tailing slash break backward compatibility? Is frontend using this API?

Reply:
turns out that endpoint is no use before(not sure why we create two end point for add tag), and I try to keep it the same as others.